src/json_rpc/websocket_server.rs

```diff
@@ -148,7 +148,7 @@ pub struct Config {
 #[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
 pub struct ConnectionId(usize);
 
-/// WebSockets listening socket and list of open connections.
+/// WebSocket's listening socket and list of open connections.
```

Comment:
```suggestion
/// Listening socket and list of open connections.
```

It's the plural of "WebSocket". "WebSocket's" doesn't really make sense